code/src/index.css

```diff
@@ -1,13 +1,164 @@
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+  }
+  
+  body {
+    margin: 0;
+    padding: 10px;
+    font-family: -apple-system, BlinkMacSystemFont, "Helvetica", sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-color: #000000;
+  }
+  
+  a {
+    text-decoration: none;
+  }
+
+  /* Header */
+  .header {
+    width: 100%;
+    font-size: 16px;
+    margin: 20px 0 15px;
+    padding-bottom: 8px;
+    border-bottom: 0.1px solid #34363A;
+  }
+  
+  .header,
+  .album-name {
+    color: #ffffff;
+  }
+
+  /* Container for all albums */
+  .site-container {
+    width: 85%;
+    max-width: 1280px;
+    margin: auto;
+  }
+  
+  .album-container {
+    margin: 1em auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  /* The cards for each album containing album-image, album-name and artist(s)-name */
+  .album-name {
+    font-size: 14px;
+    font-weight: bold;
+    padding-top: 6px;
+    margin: 6px 0;
+  }
+  
+  .artist-name {
+    color: #a0a0a0;
+    font-size: 14px;
+    padding-top: 4px;
+    margin: 0;
+  }
+  
+  .album-name:hover,
+  .artist-name:hover {
+    text-decoration-line: underline;
+    -webkit-text-decoration-line: underline;
+  }
+
+  /* Adds a comma and &-sign on right place + styling of it  */
+    .artists-links:not(:last-child):not(:nth-last-child(2))::after {
+    content: ', ';
+    color: #a0a0a0;
+  }
+  
+  .artists-links:not(:first-child):last-child::before {
+    content: ' & ';
+    color: #a0a0a0;
+  }
+  
+  .album-card {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    margin: 15px;
+    cursor: pointer;
+  }
+  
+  .album-cover {
+    display: flex;
+    position: relative;
+    background-color: black;
+  }
+  
+  .album-image {
+    display: block;
+    width: 100%;
+    transition: opacity 0.5s ease-in-out;
+  }
+  
+  .album-cover:hover .album-image {
+    opacity: 0.4;
+  }
+
+  /* Icons */
+  .icons-container {
+    display: none;
+    justify-content: space-evenly;
+    align-items: center;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    margin: 0;
+  }
+  
+  .album-card:hover .icons-container {
+    display: flex;
+  }
+  
+  .heartIcon {
+    width: 30px;
+    filter: invert(1);
+  }
+  
+  .playIcon {
+    width: 50px;
+    filter: invert(1);
+    transition: all 0.2s ease-in-out;
+  }
+  
+  .playIcon:hover {
+    transform: scale(1.3);
+    transition: all 0.3s ease-in-out;
+  }
+  
+  .dotsIcon {
+    width: 30px;
+    filter: invert(1);
+  }
+
+  /* Media queries*/
+  @media (min-width: 668px) { /* Tablet device */
+    .album-container {
+      /*Wrapping on rows instead*/
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+
+    .album-card {
+      width: 40%;
+      height: 350px;
+    }
+  }
```

Comment:
When the screen is around 1000px wide:

<img width="804" alt="Skärmavbild 2020-10-23 kl  17 10 07" src="https://user-images.githubusercontent.com/51529201/97021118-b39f5f80-1552-11eb-8847-54225da43f22.png">